packages/fxa-payments-server/src/components/TermsAndPrivacy/index.tsx

```diff
@@ -46,7 +51,7 @@ export const TermsAndPrivacy = ({ plan }: TermsAndPrivacyProps) => {
             rel="noopener noreferrer"
             target="_blank"
             data-testid="terms-download"
-            href={termsOfServiceDownloadURL}
+            href={tosUrl}
```

Comment:
I don't know the answer as I was not involved in any discussion around it.

> Per FXA-3199, we want to add the Privacy Policy download URL to the product metadata as well.

We'd still want that since we need that for the emails.